File: Python/AD-HOC/1192 - O jogo Matemático de Paula.py
"""
Autor: Pierre Vieira
Data da submissão: 26/01/2020 15:38:27
"""
n = int(input())
for c in range(n):
    line = input()
    n1, letra, n2 = int(line[0]), line[1], int(line[2])
    if n1 == n2:
        print(n1*n2)
    elif letra.islower():
        print(n1 + n2)
    else:
        print(n2 - n1)

File: Kotlin/AD - HOC/1192 - O jogo Matemático de Paula.kt
/*
* Autor: Pierre Vieira
* Data de submissão: 24/04/2021 16:57:09
*/
fun main(args: Array<String>) {
    val n = readLine()!!.toInt()
    for (i in 1..n) {
        val line = readLine()!!.toCharArray()
        val n1 = line[0].toString().toInt()
        val letra = line[1]
        val n2 = line[2].toString().toInt()
        println(
            when {
                n1 == n2 -> n1 * n2
                letra.toLowerCase() == letra -> n1 + n2
                else -> n2 - n1
            }
        )
    }
}
